src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -179,47 +183,123 @@ class ExternalDatabaseManagementService(
         return columnNamesByTableName
     }
 
+    /*UPDATE*/
+    fun updateOrganizationExternalDatabaseTable(orgId: UUID, tableFqnToId: Pair<String, UUID>, update: MetadataUpdate) {
+        update.name.ifPresent {
+            val newTableFqn = FullQualifiedName(orgId.toString(), it)
+            val oldTableName = getNameFromFqnString(tableFqnToId.first)
+            val dbName = PostgresDatabases.buildOrganizationDatabaseName(orgId)
+            assemblerConnectionManager.connect(dbName).let { dataSource ->
```

Comment:
just curious, why do we use `assemblerConnectionManager.connect(dbName)` instead of `hds` in these methods?

Reply:
assemblerConnectionManager connects to atlas, which is where these tables are stored.